app/components/supplementary-file-browser.js

```diff
@@ -0,0 +1,62 @@
+import Ember from 'ember';
+import $ from 'jquery';
+
+export default Ember.Component.extend({
+    elementId: 'preprint-file-view',
+    startValue: 0,
+    scrollAnim: '',
+    numShowing: 6,
+    selectedFile: {},
+
+    fullList: Ember.computed('files', 'files.[]', 'primaryFile', function(){
+        //Returns the list with primaryFile moved to the front
+        const files = this.get('files');
+        if(files){
+            const primaryFile = this.get('primaryFile');
+            files.splice(files.indexOf(primaryFile, 1));
+            files.unshift(primaryFile);
+        }
+        return files;
+    }),
+    supplementList: Ember.computed('fullList', 'fullList.[]', 'startValue', 'numShowing', function(){
+        if(this.get('fullList')){
+            return this.get('fullList').slice(this.get('startValue'), this.get('startValue') + this.get('numShowing'));
+        }
+    }),
+    init: function(){
+        this._super(...arguments);
+        this.set('selectedFile', this.get('primaryFile'));
+    },
+    didInsertElement: function () {
+        $('[data-toggle="tooltip"]').tooltip({});
```

Comment:
In general, we're trying to avoid embedding jquery selectors / presentation logic in the javascript controller, when possible. 

Can we do this with markup instead? There is a library called `hint-css` that Sam and I discussed evaluating for ember-osf, but haven't had a chance to look at. I'd be interested to hear your thoughts on it, as we're not 100% wed to porting over all the current OSF tooltip styling code to ember.